### DISCUSSION TITLE
Why is `silent mode` called that, instead of `dry-run mode` or `no pr creation mode`?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#28578] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

None

### Please tell us more about your question or problem

@oscr asked in https://github.com/renovatebot/renovate/issues/28154#issuecomment-2022539929

> Why is `silent mode` called silent mode? Instead of e g `dry-run mode` or `no pr creation mode`?

The [Renovate docs, onboarding behavior section](https://docs.renovatebot.com/config-overview/#onboarding-behavior) explains _why_ we have a silent mode:

> ### Onboarding behavior
> 
> If an Organization installed Renovate with "All repositories" (instead of "Selected repositories"), then Renovate will default to "Silent" mode (`dryRun=lookup`).
> We chose this behavior because:
> 
> - Too often an account or org administrator selects the "All repositories" option and accidentally onboards hundreds of repositories, and
> - By offering this option, it means that org administrators _can_ install Renovate into "All repositories" without worrying about the noise, and then let individual Repository admins decide if/when to start onboarding
> 
> If Renovate is installed, and you can see a job log, but Renovate is not onboarding your repository, look for `dryRun` in the logs to confirm you are in Silent mode and then change to Interactive mode either at the Repository level or Organization level.
> 
> Additionally, if an Organization is installed with "Selected repositories" then the app will change `onboardingNoDeps` to `true` so that an Onboarding PR is created even if no dependencies are detected.

But the docs do _not_ explain why it's called `silent mode`, instead of something like `dry-run mode` or `no pr creation mode`. Maybe some platform like GitHub/Gitlab, uses the `silent mode` term, and we copied that for consistency with the platform option name?

It's probably a good idea to either:

- rename the option
- put in the docs why the option is called silent mode

**Comment by rarkins** [comment, marked as answer]
It's because:
- It's not just no PRs, it's also no Issues
- It's a common term across other Mend capabilities, such as OSS security and SAST security, where status checks are also part of silent/non-silent

**Reply by HonkingGoose** [reply]
Trying to fix the docs with PR:

- #28579

**Reply by intgr** [reply]
It's still unclear to how the "silent" option works.

If I only want to enable Renovate in select repositories and I pick the "silent" option: will this be overridden by merely creating a `renovate.json` file -- or do I need to add some additional configuration to fully enable Renovate?

**Reply by RahulGautamSingh** [reply]
Pick the `silent` option and create a `renovate.json` with this present `mode: 'full'` in repos you want to go `full`.

**Reply by jamietanna** [reply]
@intgr can you please raise a fresh Discussion? We try to avoid "bumping" old Discussions :)
